### DISCUSSION TITLE
New Relic node.js agent in Nuxt 3

### DISCUSSION THREAD

**Discussion by memic84** [nuxt/nuxt#20893] [Questions] [answered]
Has anybody successfully implemented Nuxt 3 in New Relic APM?

We are trying to install the Node.js agent, and followed every step exactly, but get the following error: `Could not detect the install. This may be an unsupported environment.`

We are using docker containers, and the application is started with `node -r newrelic .output/server/index.mjs`

Could this be a Nuxt/Nitro unsupported thing, or just misconfiguration?

**Comment by ronenteva** [comment]
Did you figure it out?

**Reply by memic84** [reply, marked as answer]
I believe that we had forgot to add newrelic to our package.json, and needed to wait for the data to come through.

**Reply by ronenteva** [reply]
Do you mind sharing an example of how you implemented it?

**Reply by memic84** [reply]
Apologies for the late reply. We've added the newrelic package to our devDependency,
and start the server with: `node -r newrelic .output/server/index.mjs`

Assuming that you followed the other instructions from New Relic, for the infrastructure part.

**Comment by luksak** [comment]
I just published https://github.com/attributech/nuxt-newrelic that provides better integration between Nuxt and New Relic. This is my first Nuxt module and my first npm package. I'd be happy to receive some feedback on it 🫶
